src/store/store.js

```diff
@@ -1,12 +1,13 @@
-import { applyMiddleware, createStore, compose } from 'redux';
+import {applyMiddleware, compose, createStore} from 'redux';
 import thunk from 'redux-thunk';
 import persistState from 'redux-localstorage'
 import rootReducer from '../reducers';
 
-const enhancements = compose(
+const composer = (process.env.NODE_ENV === 'development' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose;
```

Comment:
Support for [redux dev tools chrome extension](https://github.com/zalmoxisus/redux-devtools-extension#usage).